secure/storage/src/vault.rs

```diff
@@ -246,7 +246,7 @@ impl KVStorage for VaultStorage {
     }
 
     fn set(&mut self, key: &str, value: Value) -> Result<(), Error> {
-        // Vault internally does not distinguish create versus udpate except by permissions. So we
+        // Vault internally does not distinguish create versus update except by permissions. So we
```

Comment:
I actually delete this in my PR set... so if you refactor this before landing, feel free to strip this.